webapp/src/shared/components/Input/Input.tsx

```diff
@@ -0,0 +1,41 @@
+import { Color } from '../../types';
+import Icon, { IconSize, IconVariant } from '../Icon/Icon';
+import './Input.css';
+
+export enum InputColor {
+  LIGHT = 'light',
```

Comment:
I'm not sure why do we need this class. We already have a enum for colors. I see that you want to put logic on top in order to transform the color from the prop into different colors, but I don't understand why would we need a special separate symbol. Also, it's ambiguous. Input color? Color of the text that is the input or of the background of the input text? Or of the component named 'input' that contains both? It's confusing.

Reply:
What do we think about naming it to `InputVariant`?

Reply:
Yes, again I tried to follow the issue scheme, but you are right, we can rename it to something better and try to take advantage of the enum color we already have. I will have a look